storage_devices/lvm.py

```diff
@@ -498,22 +504,40 @@ def remove_pv(pv_name: str):
         cmd = f"pvremove {pv_name}"
         return TestRun.executor.run(cmd)
 
+    @staticmethod
+    def get_os_vg():
+        os_vgs = []
+        disks = get_system_disks()
+        for disk in disks:
+            cmd = (rf"pvdisplay --select pv_name=~\/dev\/{disk} | "
+                   rf"grep 'VG Name' | awk '{{print $3}}'")
+            os_vg_name = TestRun.executor.run(cmd).stdout
+            if os_vg_name:
+                os_vgs.append(os_vg_name)
+        return os_vgs
+
```

Comment:
My suggestion is to use pvscan instead of pvdisplay and get all needed info from one command
```suggestion
    @staticmethod
    def get_os_vg():
        disks = get_system_disks()
        cmd = ("pvscan | grep -e /dev" + " -e /dev/".join(disks) + " | awk '{{print $4}}'")
        os_vg_names = TestRun.executor.run(cmd).stdout
        if os_vg_names:
            return set(os_vg_names.split("\n"))  # remove duplicates
        return []

```

Reply:
you can also remove duplicates directly in awk `'$2 != "" && !seen[$2]++ {print $2}'`, but it's less intuitive

Reply:
I also think it will make the command less readable and harder to understand